llmfoundry/data/finetuning/dataloader.py

```diff
@@ -683,6 +686,8 @@ def build_collate_fn(
         'timeout': 0,
     })
 
+    mosaicml_logger = maybe_create_mosaicml_logger()
```

Comment:
idk if this is necessary, given that this exception was already logged somewhere. is this caught and logged somewhere else (similar to `convert_text_to_mds`)?

@jjanezhang